contracts/staking/utils/StakingRewardsPerEpoch.sol

```diff
@@ -13,37 +13,34 @@ contract StakingRewardsPerEpoch is ReentrancyGuard, Pausable {
 	struct StakerInfo {
 		uint256 reward; // the reward amount which should be transfered to the user
 		uint256 balance; // the amount of active stake of user
-		uint256 pendingStake; // the amount of pending stake of user
-		uint256 indexOfLastEpochStaked; // an index of the epoch from which the reward is calculated
+		uint256 rewardPerTokenPaid; // the amount of pending stake of user
```

Comment:
this is the amount already paid, not pending

Reply:
Yes, I'll edit the comments of where I changed now